ras_party/controllers/party_controller.py

```diff
@@ -21,8 +21,10 @@ def parties_post(party_data, session):
     :param party_data: packet containing the data to post
     :type party_data: JSON data maching the schema described in schemas/party_schema.json
     """
+    logger.info("party data in controller", party_data=party_data)
```

Comment:
Does this have personal data in it?

Reply:
Overhang of debugging code?

Reply:
It was yeah.  Sorry for wasting your guys time with dumb stuff in the PR >_<